src/50_Sample_AMPs/Recipe.html

```diff
@@ -0,0 +1,466 @@
+      "prepTime": "PT30M",
+      "cookTime": "PT1H45M",
+      "totalTime": "PT2H15M",
+      "recipeYield": "1 tray (4 servings)",
+      "nutrition": {
+        "@type": "NutritionInformation",
+        "servingSize": "1 piece",
+        "calories": "500 kcal",
+        "carbohydrateContent": "5g",
+        "fatContent": "30g",
+        "proteinContent": "60g"
+      },
+      "recipeIngredient": [
+        "lamb",
+        "4 cloves of garlic",
+        "1 sprig of rosemary",
+        "4 chopped tomatoes",
+        "1 tbsp olive oil",
+        "1 carrot",
+        "1 cup wine",
+        "peppercorn to taste"
+      ],
+      "recipeInstructions": "1. Heat oven to 350F / 180C.\n2. Prepare the meat by washing and removing the silverskin.\n3. Peel and cut each garlic clove into several slices.\n4. Make incisions in the meat and insert the garlic slices.\n5. Place lamb in baking dish, drizzle with olive oil, and add wine.\n6. Add chopped tomatoes and carrot.\n7. Cut lemon into quarters, remove seeds, squeeze juice over meat and leave wedges among the meat pieces.\n8. Finely mince rosemary and sprinkle over lamb, along with paprika, salt and peppercorns to taste.\n9. Roast for approximately 1 hr 45 mins, turning the lamb halfway through.",
+      "publisher": {
+        "@type": "Organization",
+        "name": "Google",
+        "logo": {
+          "@type": "ImageObject",
+          "url": "https://www.google.com/images/branding/googlelogo/1x/googlelogo_color_116x41dp.png",
+          "width": 116,
+          "height": 41
+        }
+      },
+      "aggregateRating": {
+        "@type": "AggregateRating",
+        "ratingValue": "4.5",
+        "bestRating": "5",
+        "ratingCount": "3",
+        "reviewCount": "3"
+      },
+      "review": [{
+        "@type": "Review",
+        "name": "Yum!",
+        "reviewBody": "Words cannot express how delicious this tastes!",
+        "reviewRating": {
+          "@type": "Rating",
+          "ratingValue": "5"
+        },
+        "author": "Positive Patrick"
+      }, {
+        "@type": "Review",
+        "name": "Not bad, but prep is laborious",
+        "reviewBody": "Tastes really good, but cutting the silverskin takes a while. Try asking your butcher if they can do it, since they have specialized tools and skills!",
+        "reviewRating": {
+          "@type": "Rating",
+          "ratingValue": "4"
+        },
+        "author": "Negative Nancy"
+      }, {
+        "@type": "Review",
+        "name": "Unexpected side effects",
+        "reviewBody": "After taking it out of the oven, I left this dish to cool off next to my pet lizard, unfortunately now he's 350ft tall now and is currently destroying Tokyo, Japan.",
+        "reviewRating": {
+          "@type": "Rating",
+          "ratingValue": "4"
+        },
+        "author": "Lawrence Gonzalez"
+      }]
+    }
+  </script>
+
+
+  <!-- [Responsive `meta` tag](https://css-tricks.com/snippets/html/responsive-meta-tag/). -->
```

Comment:
You don't need to explain the AMP basics, these are covered elsewhere. Focus only on the recipe specific bits.